tools/python_api/src_cpp/include/py_query_result.h

```diff
@@ -6,10 +6,12 @@
 #include "arrow_array.h"
 #include "common/arrow/arrow.h"
 #include "common/types/internal_id_t.h"
+#include "common/types/value/value.h"
 #include "main/kuzu.h"
 #include "pybind_include.h"
 
 using namespace kuzu::main;
+using namespace kuzu::common;
```

Comment:
This probably doesn't matter much, but we usually avoid using namespace in header files because it will be propagated through other includes. I would remove `using namespace kuzu::common`